docs/source/reference/numpysupported.rst

```diff
@@ -279,7 +279,7 @@ floating-point and complex numbers:
 * :func:`numpy.linalg.svd` (only the 2 first arguments).
 
 .. note::
-   The implementation of these functions needs Scipy 0.16+ to be installed.
+   The implementation of these functions needs Scipy to be installed.
```

Comment:
while we are here, should caps this as `SciPy`

Reply:
fixed